Dockerfile

```
@@ -1,31 +1,40 @@
-FROM quay.io/opendatahub/text-generation-inference:stable-bafd218
-
-USER root
+ARG BASE_UBI_IMAGE_TAG=8.8-1067
+FROM registry.access.redhat.com/ubi8/ubi:${BASE_UBI_IMAGE_TAG} as base
 
 # Add grpc-ecosystem health probe
 ARG GRPC_HEALTH_PROBE_VERSION=v0.4.20
 
-WORKDIR /caikit
-COPY caikit /caikit
-
-RUN yum -y update && yum -y install git git-lfs && yum clean all && \
-    git lfs install && \
-    pip install 'micropipenv[toml]' && \
-    micropipenv install && \
-    rm -rf ~/.cache && \
-    mkdir -p /opt/models && \
-    adduser -g 0 -u 1001 caikit --home-dir /caikit && \
-    chown -R 1001:0 /caikit /opt/models && \
-    chmod -R g=u /caikit /opt/models
-
 # This is for the use-cases without kserve
 RUN curl -Lo /usr/local/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
     chmod +x /usr/local/bin/grpc_health_probe
```

Would it be possible to determine the CPU arch at build time and use the appropiate package? On arm64 this seems to be the right package.
```suggestion
RUN curl -Lo /usr/local/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-arm64 && \
```

---

@Xaenalt  how about getting rid of this?

---

Yeah, we should take that down as a note for when we go to multiarch builds, but yeah separate PR for now is fine